examples/synchronization/multi_function/cont_ai_ci_tdms_sync.py

```diff
@@ -0,0 +1,208 @@
+"""Example of logging multiple synchronized tasks to a single TDMS file using a queue.
+
+This example demonstrates how to:
+1. Synchronize multiple DAQmx tasks using a shared clock
+2. Use a producer-consumer pattern with a queue
+3. Log data from multiple tasks to a single TDMS file
+"""
+
+import os
+import queue
+import threading
+import time
+from typing import List
```

Comment:
typing.List has been deprecated since we dropped Python 3.8 support. Use `list`, or even better, use `collections.abc.Sequence`.